docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -583,50 +583,46 @@ $ curl -H "Authorization: Bearer eyJraWQ..." http://127.0.0.1:8080/secured/roles
 hello jdoe@quarkus.io, isHttps: false, authScheme: Bearer, hasJWT: true, birthdate: 2001-07-13
 ----
 
-=== Package and run the application
+=== Run the application in JVM mode
 
-As usual, the application can be packaged by using:
+You can run the application as a standard Java application.
 
+. Compile the application:
++
+====
 include::{includes}/devtools/build.adoc[]
-
-And executed by using `java -jar target/quarkus-app/quarkus-run.jar`:
-
-.Runner jar example
-[source,shell,subs=attributes+]
+====
```

Comment:
Hi Sergey,

The admonition blocks (`====`) is generally used for admontions such as `NOTE` and `IMPORTANT`.
I think you could use open blocks (`--`) here instead.

Rolfe

Reply:
https://docs.asciidoctor.org/asciidoc/latest/blocks/open-blocks/

Reply:
Hi Rolfe, I don't mind at all, just would like to note it is a copy and paste from the https://quarkus.io/guides/security-oidc-bearer-token-authentication-tutorial#run-the-application-in-jvm-mode, so it currently will look here as in that doc... May be it is worth doing a sep PR to align it across multiple docs ? Sorry if I've missed something

Reply:
You haven't missed anything, it's common for these types of things to self-propagate this way. 
Though I'd like to help find and fix this elsewhere, I must remain focused on other higher priorities right now.

Reply:
>You haven't missed anything, it's common for these types of things to self-propagate this way.

Definitely :-), so I guess, given your thumbs up in the prev comment, we can leave it as is for now ?

Reply:
Np at all, if you prefer starting fixing it from this PR

Reply:
Thanks @rolfedh, ok, I'll deal at some point later as I'm also off Fri and Mon...